elastic/kibana
# [share] hide 'Embed code' share option when buildFlavor is 'serverless'

Closes https://github.com/elastic/kibana/issues/161266

### Overview
PR hides 'Embed code' share option in serverless

### Test
* Start kibana with `yarn start --serverless=es`
* verify 'Embed code' is not displayed in Dashboard share option
